src/Integrator/Base/Mechanics.H

```diff
@@ -142,6 +146,8 @@ protected:
             if (ic_rhs) ic_rhs->Initialize(lev, rhs_mf);
         }
 
+        UpdateModel(a_step);
```

Comment:
We need to determine why `UpdateModel` is called twice. Is it necessary? If so, we need to add an option to enable or disable `UpdateModel` before and after initialization.